chart/values.yaml

```diff
@@ -435,6 +435,22 @@ cleanStatsCache:
   expiredTimeIntervalSeconds: 600
   subfolderPattern: "*"
 
+deleteObsoleteCache:
+  enabled: true
+  log:
+    level: "info"
+  action: "delete-obsolete-cache"
+  error_codes_to_retry: ""
```

Comment:
is this required?

Reply:
No, I removed it in the other jobs as well.